app/views/homepage/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class=row>
+  <div class="col-md-4">
+    <h1>Top Movies</h1>
+    <% @movies.each do |movie| %>
+      <h2><%= link_to movie.title, movie_path(movie) %><small> Upvotes: <%= movie.upvotes %></small></%></h2>
```

Comment:
Whoops – extra mistyped ERB tags `</%>` on this file. Not a biggie, but keep an eye out for little things like this.